src/error.ts

```diff
@@ -14,110 +14,109 @@ export enum codes {
   ERR_TOO_MANY_OUTBOUND_PROTOCOL_STREAMS = 'ERR_TOO_MANY_OUTBOUND_PROTOCOL_STREAMS',
 }
 
-export class WebRTCTransportError extends Error {
-  constructor (msg: string) {
-    super(`WebRTC transport error: ${msg}`)
+export class WebRTCTransportError extends CodeError {
+  constructor (msg: string, code: string) {
+    super(`WebRTC transport error: ${msg}`, code)
     this.name = 'WebRTCTransportError'
   }
 }
```

Comment:
Other breaking change

Reply:
Can we avoid the breaking change by making the parameter optional?

Reply:
empty string fine?